Resources/Prototypes/Entities/Clothing/OuterClothing/hardsuits.yml

```diff
@@ -32,8 +32,8 @@
 - type: entity
   parent: ClothingOuterHardsuitBase
   id: ClothingOuterHardsuitCap
-  name: captain's armor
-  description: A suit of protective formal armor made for the station's captain.
+  name: captain's armored spacesuit
```

Comment:
Fairly sure its intentional to be unarmored

Reply:
> Fairly sure its intentional to be unarmored

It is an armoured spacesuit as it protects against sharp and blunt, just not very well. I could rename it to captain's spacesuit, but the discussing I had yesterday ended up on this name.